src/services/item/plugins/action/service.ts

```diff
@@ -207,12 +207,12 @@ export class ActionItemService {
         actions: AppAction[];
       };
     } = {};
-    for (const { id: appId } of [item, ...descendants].filter(
-      ({ type }) => type === ItemType.APP,
-    )) {
+    const appItems = [item, ...descendants].filter(({ type }) => type === ItemType.APP);
+    for (const { id: appId } of appItems) {
       const appData = await repositories.appDataRepository.getForItem(
         appId,
         {},
+        // needs investigating: does this mean a reader could export the actions of an item and see all users responses ?
```

Comment:
Only admin should be allowed to do that right?

Reply:
Add an issue for this.

Reply:
Yes, we cannot allow a reader to get access to other users' data with the export. We may give them the possibility to export their own data.